lib/identity_doc_auth/mock/result_response_builder.rb

```diff
@@ -91,6 +110,44 @@ def pii_from_doc
       def success?
         errors.blank?
       end
+
+      DEFAULT_FAILED_ALERTS = [{ name: '2D Barcode Read', result: 'Failed' }].freeze
+      DEFAULT_IMAGE_METRICS = {
+        front: {
+          "VerticalResolution" => 300,
+          "HorizontalResolution" => 300,
+          "GlareMetric" => 50,
+          "SharpnessMetric" => 50,
+        },
+        back: {
+          "VerticalResolution" => 300,
+          "HorizontalResolution" => 300,
+          "GlareMetric" => 50,
+          "SharpnessMetric" => 50,
+        }
+      }.freeze
+
+      def create_response_info(
+        doc_auth_result: 'Failed',
```

Comment:
We never pass this argument. Are we expecting to? Should we just hard-code it into the return value?

Reply:
ah! I was going to add that as an option but forgot to add the line. I'll add it.